### PR TITLE
Commands will use resolved directory instead of package name

### PR DESCRIPTION
**Pull request by pomek**
Fixes: #25.

### CODE REVIEW

File: lib/commands/bootstrap.js [hunk 1]
```diff
@@ -15,8 +15,11 @@ module.exports = {
 	 * @param {Object} data
 	 * @param {Object} data.options Additional options provided by the user.
 	 * @param {String} data.options.recursive Whether to install dependencies of packages.
-	 * @param {String} data.options.repositoryResolver Module which will resolve repositories for packages.
 	 * @param {String} data.name Name of current package that will be parsed.
+	 * @param {Object|null} data.repository
```
Comment (Reinmar) [inline lib/commands/bootstrap.js]: Please extract this to some typedef. Repeating documentation, as well as the code, is bad.

You can put that typedef in `parserepositoryurl.js`.

File: lib/commands/exec.js [hunk 2]
```diff
@@ -32,7 +36,14 @@ module.exports = {
 		const log = require( '../utils/log' )();
 
 		return new Promise( ( resolve, reject ) => {
-			const newCwd = path.join( data.options.cwd, data.mgit.packages, data.name );
+			// The repository was not found so the package is skipped.
+			if ( !data.repository ) {
```
Comment (Reinmar) [inline lib/commands/exec.js]: This log repeats in every command now.

Comment (pomek) [inline lib/commands/exec.js]: Any solution? ;)

Comment (Reinmar) [inline lib/commands/exec.js]: Moving it out to `lib/index.js`?

Comment (Reinmar) [inline lib/commands/exec.js]: Why running a child process if repository could not be resolved anyway?

Comment (pomek) [inline lib/commands/exec.js]: > Moving it out to `lib/index.js`?

I found a solution which allows using current logs and does not execute the command. See: 799feda.

**Pull request merged**